src/functionalTest/java/uk/gov/hmcts/darts/courthouses/CourthousesFunctionalTest.java

```diff
@@ -162,21 +162,6 @@ void getCourthouseIdDoesNotExist() {
 
     @Test
     @Order(8)
-    void deleteCourthouse() {
-        Response response = buildRequestWithExternalAuth()
-            .contentType(ContentType.JSON)
-            .when()
-            .baseUri(getUri(COURTHOUSES_URI + "/" + testCourthouseId))
-            .delete()
-            .then()
-            .statusCode(NO_CONTENT)
-            .extract().response();
-
-        assertNotNull(response);
```

Comment:
State leaks between the Courthouse functional tests, so this is sadly a crucial step because its role is to also clean up the courthouse that was created in a preecding test. So when removing this test, we need to rethink how this test class handles clean up.

I'm somewhat working on this as part of DMP-2279, so might be worth holding this ticket until DMP-2279 is merged in.